src/libuvc/libuvc.cpp

```diff
@@ -629,9 +629,9 @@ namespace librealsense
             /* recieved a frame and call the callback. */
```

Comment:
Even though it is not part of the PR - can you fix the typo: ~~recieved~~received

Reply:
done